packages/runtime/test/integration/execute-transaction.ts

```diff
@@ -224,6 +246,284 @@ describe("Algorand Smart Contracts - Execute transaction", function () {
 			];
 
 			runtime.executeTx(tx);
+			syncAccounts();
+			// verify
+			assert.exists(alice.getAppFromLocal(appInfo?.appID))
+		}
+	});
+
+	it("Should opt-in and call app, through execute transaction", () => {
```

Comment:
Line of codes in 231 and 255 are pretty much the same, is this one duplicated?

Reply:
Duplicated by mistake. Removed it.